src/controller/editingcontroller.js

```diff
@@ -23,6 +23,7 @@ import {
 } from '../conversion/model-selection-to-view-converters';
 
 import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import '../../theme/theme.scss';
```

Comment:
Well, there are important things to remember here.

Importing an SCSS file many times may lead to code duplication and bugs. Even if `placeholder.scss` will be imported only once, it may be importing other SCSS files which are imported somewhere else already, making those rules appear twice. This is super sad and dangerous. I hit this hard when trying to start importing SCSS files in their proper JS components. It blew up.

The solution here is a refactoring of our SCSS. The code needs to be split into two layers – helpers and components. Helpers must not contain any rules – they should only contain mixins, functions, etc. This means that you will be able to import a helper multiple times without duplicating the code. The components will use helpers to define final CSS rules. However, a single component must not be imported more than once.

The second thing is that we must support multiple themes. No one so far made any research about that and the only related ticket so far is https://github.com/ckeditor/ckeditor5-ui/issues/144. But it clarifies that plugins (and engine) must not depend on a particular theme. This is a long topic and it doesn't make sense to spend time on it now, but how it applies to this particular case? It doesn't. The import can be anywhere. But the SCSS file should, at least for now, have no dependency on the theme-lark package.